src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String line = "\t________________________________________";
+    public static Task[] tasks = new Task[100];
+    public static int currentTask = 0;
+
+    public static void takeInput() {
+        Scanner scannerObject = new Scanner(System.in);
+        while (scannerObject.hasNextLine()) {
+            String command = scannerObject.nextLine();
+            if (command.equals("bye")) {
+                bye();
+                break;
+            } else if (command.equals("list")) {
+                list();
+            } else if (command.startsWith("done")) {
+                done(command.split(" ")[1]);
+            } else if (command.startsWith("todo")) {
+                addTodo(command.split(" ", 2)[1]);
+            } else if (command.startsWith("event")) {
+                addEvent(command.split(" ", 2)[1]);
+            } else if (command.startsWith("deadline")) {
+                addDeadline(command.split(" ", 2)[1]);
+            }
+        }
+    }
+
+    public static void bye() {
+        System.out.println(line);
+        System.out.println("\tBye. Hope to see you again soon!");
+        System.out.println(line + "\n");
+    }
+
+    public static void list() {
+        System.out.println(line);
+        System.out.println("\tHere are the tasks in your list:");
+        for (int i=0; i<currentTask; i++) {
+            System.out.println("\t" + Integer.toString(i+1) + ".[" + tasks[i].getType() + "]["
+                    + tasks[i].getStatusIcon() + "] " + tasks[i].getDescription() + " " + tasks[i].getDate());
+        }
+        System.out.println(line + "\n");
+    }
+
+    public static void addTodo(String todo) {
+        Todo t = new Todo(todo);
```

Comment:
a different variable name could be used instead of t, may lead to confusion further along the code